pom.xml

```diff
@@ -242,21 +237,9 @@
 		</dependency>
 		<dependency>
 			<groupId>com.github.tomakehurst</groupId>
-			<artifactId>wiremock-jre8</artifactId>
+			<artifactId>wiremock-jre8-standalone</artifactId>
```

Comment:
always better to use the shaded wiremock as this has jetty and guava dependencies which conflict with Jenkins' version